README.md

```diff
@@ -1 +1,180 @@
-# First-Capstone
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone_Project_1] <a name="about-project"></a>
+
+**Capstone project** is a project based learning activity that is carried out at the end of evry module.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+<summary>Javascript</summary>
+</details>
+
+> Describe between 1-3 key features of the application.
+
+- Html and css best practices.
+- Javascript best prectices
+- Use of DOM to manipulate javascript objects
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://google.com): To be included.
```

Comment:
- [ ] The link to the live demo of your project leads to a google page, I'll suggest you correctly deploy your project with Githubpages and add the correct link here. :+1: :+1:

Reply:
Good job implementing this change.